functions/__fzf_search_history.fish

```diff
@@ -6,7 +6,7 @@ function __fzf_search_history --description "Search command history. Replace the
         builtin history --null --show-time="%m-%d %H:%M:%S | " |
         fzf --read0 --tiebreak=index --query=(commandline) \
             # preview current command in a window at the bottom 3 lines tall
-            --preview="echo {4..}" \
+            --preview="echo -- {4..} | bat --plain --color=always --language=fish" \
```

Comment:
I noticed --plain disables pager but then I tested it anyway and the pager is still used for long commands. What gives? 🤔

Reply:
`--plain` doesn't disable pager, AFAICT.